README.md

```diff
@@ -18,21 +18,23 @@ Orion is a C++ implementation of the NGSI9/10 REST API binding developed as a pa
 
 Orion Context Broker allows you to manage all the whole lifecycle of context information including updates, queries, registrations and subscriptions. It is an NGSI9/10 server implementation to manage context information and its availability. Using the Orion Context Broker, you are able to register context elements and manage them through updates and queries. In addition, you can subscribe to context information so when some condition occurs (e.g. an interval of time has passed or the context elements have changed) you receive a notification. These usage scenarios and the Orion Context Broker features are described in this document.
 
-If this is your first approach with Orion Context Broker, it is highly recommended to have a look to the brief [Quick Start guide](doc/md/quick_start_guide.md).
+If this is your first approach with Orion Context Broker, it is highly recommended to have a look to the brief [Quick Start guide](doc/manuals/quick_start_guide.md).
 
 ## Build and Install
 
 The recommended procedure is to install using RPM packages in CentOS 6.x. If you are interested in
-building from sources, check [this document](doc/md/admin/build_source.md).
+building from sources, check [this document](doc/manuals/admin/build_source.md).
 
 ### Requirements
 
-* System resources: see [these recommendations](doc/md/admin/resources.md#resources-recommendations)
+* System resources: see [these recommendations](doc/manuals/admin/resources.md#resources-recommendations)
 * Operating system: CentOS/RedHat. The reference operating system is CentOS 6.3  
 but it should work also in any later CentOS/RedHat 6.x version.
-* Database: MongoDB is required running either in the same system where Orion Context Broker is going to be installed or in a different host accessible through the network. The recommended MongoDB version is 2.6.9 (although it should work with later MongoDB 2.6.x and 3.0.x versions). It is not recommended using MongoDB 2.4.x., as some [geolocated queries](doc/md/user/geolocation.md).
+* Database: MongoDB is required running either in the same system where Orion Context Broker is going to be installed or in a different host accessible through the network. The recommended MongoDB version is 2.6.9 (although it should work with later MongoDB 2.6.x and 3.0.x versions). It is not recommended using MongoDB 2.4.x., as some [geolocated queries](doc/manuals/user/geolocation.md) may not work.
```

Comment:
English:
 * is required to run
 * It is not recommended to use MongoDB 2.4.x ...

Actually, I would like to rephrase the first sentence like this:
*Database: MongoDB is required to run either in the same host where Orion Context Broker is to be installed or in a different host accessible through the network.*